website/src/components/ReviewPage/ReviewCard.tsx

```diff
@@ -48,6 +49,8 @@ export const ReviewCard: FC<ReviewCardProps> = ({
     accessToken,
 }) => {
     const { isLoading, data } = useGetMetadataAndAnnotations(organism, clientConfig, accessToken, sequenceEntryStatus);
+    // TODO the hook above makes an extra call for each row.
```

Comment:
Is there an issue that we can reference in the TODO? (Otherwise TODOs will likely never be done)

Reply:
yes there is a ticket for it, I'll just remove the TODO here: https://github.com/loculus-project/loculus/pull/3125/commits/6477e6dac5f33197c2b472d699350ecd99747ee9